platform/core/src/extensions/ExtensionManager.js

```diff
@@ -277,13 +277,16 @@ export default class ExtensionManager {
   };
 
   _initDataSourcesModule(extensionModule, extensionId, dataSources = []) {
+    const { UserAuthenticationService } = this._servicesManager.services;
```

Comment:
minor comment: does userAuthenticationService needs reset or clear upon exit as we do for these
https://bit.ly/3p9zAgt

Reply:
I tried this but then navigating between pages (i.e. viewer and worklist) requires the user to log in again.